docs/pages/transactions/using-eip-7702/third-party.mdx

```diff
@@ -0,0 +1,76 @@
+If you have an existing custom signer or another third-party embedded wallet provider, you can upgrade your embedded EOAs to our smart wallets by connecting your existing signer.
+
+**Requirements**
+Your signer or embedded wallet provider must support `signAuthorization` in order to be compatible with EIP-7702 delegations.
+To bring your own signer, you'll create a `SmartAccountSigner` interface that implements `signAuthorization`.
+See the details for the interface requirements [here](/wallets/third-party/signers#implementing-smartaccountauthenticator-or-smartaccountsigner).
+
+For example, you can upgrade an existing Privy embedded EOA by extending `WalletClientSigner` to include the `signAuthorization` method exposed by Privy.
+
+<CodeBlocks>
+
+```ts twoslash example.ts
```

Comment:
TODO: modify https://github.com/avarobinson/alchemy-wallets-7702-thirdparty-example to make sure that this example works. Consider updating the example master.